diff --git a/Makefile b/Makefile
index a117c8c1..39c8e559 100644
--- a/Makefile
+++ b/Makefile
@@ -2,6 +2,7 @@
 #
 # Tiny C Compiler Makefile
 #
+HOSTCC=cc
 
 ifndef TOP
  TOP = .
@@ -264,7 +265,7 @@ endif
 
 # convert "include/tccdefs.h" to "tccdefs_.h"
 %_.h : include/%.h conftest.c
-	$S$(CC) -DC2STR $(filter %.c,$^) -o c2str.exe && ./c2str.exe $< $@
+	$S$(HOSTCC) -DC2STR $(filter %.c,$^) -o c2str.exe && ./c2str.exe $< $@
 
 # target specific object rule
 $(X)%.o : %.c $(LIBTCC_INC)
